pkg/core/intl/oidc.go

```diff
@@ -0,0 +1,83 @@
+package intl
+
+import (
+	"fmt"
+	"sort"
+	"strings"
+
+	"golang.org/x/text/language"
+)
+
+// LocalizeOIDCJSONObject returns the localized value of key in jsonObject according to preferredLanguageTags.
+func LocalizeOIDCJSONObject(preferredLanguageTags []string, jsonObject map[string]interface{}, key string) string {
```

Comment:
I think it's better not to use 'OIDC' in the names, since we're not just using it for OIDC, but also other localized strings in configuration.